colossalai/inference/config.py

```diff
@@ -202,11 +202,12 @@ class InferenceConfig(RPC_PARAM):
     ] = 1.2  # the ratio of prefill sequences to decoding sequences, we do prefill step once the actual value exceeds ratio
     pad_input: bool = False
     early_stopping: Optional[bool] = False
-    top_k: Optional[int] = None
-    top_p: Optional[float] = None
+    top_k: Optional[int] = 50
```

Comment:
why use 50?

Reply:
Corresponds to default value of transformers